docs/hub/webhooks.md

```diff
@@ -0,0 +1,120 @@
+# Webhooks
+
+Webhooks allow you to know when new changes happen on specific repos or repos belonging to specific users / organizations (not just your repos, but any repo!).
+
+You can configure the webhooks in your [settings](https://huggingface.co/settings/webhooks).
+
+You can watch repo changes or community events. If you watch all repos belonging to a user or an organization, you can also receive events for repo creation.
```

Comment:
"receive events" => what does this mean?

Reply:
I changed the paragraph